applications/crossbar/priv/couchdb/schemas/system_config.callflow.json

```diff
@@ -64,14 +64,6 @@
             "description": "callflow post answer delay",
             "type": "integer"
         },
-        "privacy_name": {
```

Comment:
@jamesaimonetti I already did that, also if you notice I'm calling `kapps_config:migrate/1` in `kapps_controller` during start up to initiate config migration.